docs/core/linux-prerequisites.md

```diff
@@ -191,7 +191,7 @@ For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Co
 **.NET Core 2.1**
 
 >[!IMPORTANT]
-> To use .NET Core 2.1 with Visual Studio, you need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/preview).
+> To use .NET Core 2.1 with Visual Studio, you need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://visualstudio.microsoft.com/vs/preview).
```

Comment:
not related to your change, but we need to update the prereqs for 2.1 topic. would you mind changing the links in this topic to 
`[install Visual Studio 2017 15.7 or newer]](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017)`?

Reply:
Sure, I'll make the change, @mairaw.